### ISSUE TITLE
Communicate with the rower on MS Windows

### ISSUE THREAD

**Issue by phl3x0r** [phl3x0r/electron-rower#8]
We should be able to set up and communicate with the rower on a Microsoft Windows environment.

**Comment by dmamalis** [comment]
Have you tried to add my work on the communication part?

**Comment by phl3x0r** [comment]
Yes, I've tried all kinds of things, but I cannot see any output from the rower.
The app works fine, so it's not the cable or the rower.
Do you have any luck with this on windows?

**Comment by phl3x0r** [comment]
@dmamalis You can find the TS converted code [here ](https://github.com/phl3x0r/electron-rower/blob/master/src/node.ts)(previously main.js), and the readout code [here ](https://github.com/phl3x0r/electron-rower/blob/master/src/server/readout.ts)(previously readout.js)

The lines using the readout is currently commented out in node.ts
